Date: May 13 19:10:31 2014

From: 6f6a58e3c693dbb57b11cc9b78ce6c732de91ebc May 17 06:36:08 2014

Subject: [PATCH] RDKSVREQ-2193: Recipes for alljoyn and related support libraries

	Includes recipes for alljoyn core and services, and support libraries
	for implementing remote controller and controlpanel browser demos.

	The source code for the support libraries are currently stored in
	private repositories.

	The control panel changes include modifications to rdkbrowser. Those
	changes have been added as a patch file as there is no separate svn
	branch available for checking in the source code.
	
Source: COMCAST


Upstream-Status: Pending

Signed-off-by: Khem Raj <khem_raj@cable.comcast.com>
---

Index: source/core/alljoyn/SConstruct
===================================================================
--- source.orig/core/alljoyn/SConstruct
+++ source/core/alljoyn/SConstruct
@@ -21,10 +21,18 @@ env = SConscript(['build_core/SConscript
 vars = Variables()
 
 vars.Add('BINDINGS', 'Bindings to build (comma separated list): cpp, c, java, js', 'cpp,c,java,js')
-vars.Add('SERVICES', 'AllJoyn services libraries to build (comma separated list): config,controlpanel,notification,onboarding,time,audio', '')
+vars.Add('SERVICES', 'AllJoyn services libraries to build (comma separated list): config,controlpanel,notification,onboarding,audio', 'about,config,controlpanel,notification,onboarding')
+vars.Add('CC', 'C Compiler', 'gcc')
+vars.Add('CXX', 'C++ Compiler', 'g++')
+vars.Add('LINK', 'Linker', 'gcc')
+vars.Add('AR', 'Archive', 'ar')
+vars.Add('RANLIB', 'ranlib', 'ranlib')
+vars.Add('LINKFLAGS', 'Link flags', '--default-value-depends-on-cpu-arch-etc')
+env['ENV']['PATH'] = os.environ['PATH']
 vars.Add(EnumVariable('BUILD_SERVICES_SAMPLES', 'Build the services samples that require libxml2 and json libraries.', 'on', allowed_values = ['on', 'off']))
 vars.Add(BoolVariable('BUILD_DDAPI', 'Flag to indicate if data-driven API has to be built or not', 0))
 vars.Update(env)
+
 Help(vars.GenerateHelpText(env))
 
 bindings = set([ b.strip()
